.travis.yml

```diff
@@ -11,4 +11,30 @@ matrix:
 
 cache: cargo
 
-sudo: false
+sudo: true
+
+services:
+- docker
+
+install:
+- gem install fpm --no-rdoc --no-ri
+- gem install package_cloud --no-ri --no-rdoc
+
+after_success:
+- mkdir opt
+- cp target/debug/weldr opt
+# Use the same docker image as much as possible
+- docker run -v "$(pwd):/src/" cdrx/fpm-centos:7 -s dir -t deb -v 0.1.0 -n weldr -C /src  opt/weldr
```

Comment:
> >         x86_64-unknown-linux-musl
> 
> yes, looks like (ref).
> Should I use the gnu for now?

The musl build target works fine. See the [DOCKER.md](https://github.com/hjr3/weldr/blob/master/DOCKER.md). If that is not working though, we can do a normal release `cargo build --release` and store that resulting binary in the deb/rpm package.